.github/fabricbot.json

```diff
@@ -996,6 +996,46 @@
         "taskName": "Approve PRs to release branches that are approved by InfraSwat"
       },
       "disabled": true
+    },
+    {
+      "taskType": "trigger",
+      "capabilityId": "IssueResponder",
+      "subCapability": "PullRequestResponder",
+      "version": "1.0",
+      "config": {
+        "conditions": {
+          "operator": "and",
+          "operands": [
+            {
+              "name": "prMatchesPattern",
+              "parameters": {
+                "matchRegex": ".*/PublicAPI\\.(Shipped|Unshipped)\\.txt"
+              }
+            }
+          ]
+        },
+        "eventType": "pull_request",
+        "eventNames": [
+          "pull_request",
+          "issues",
+          "project_card"
+        ],
+        "taskName": "Adds \"Needs Design Review\" on PRs that touch public APIs",
+        "actions": [
+          {
+            "name": "addLabel",
+            "parameters": {
+              "label": "Need Design Review"
```

Comment:
This _might_ result in occasional confusion if this label gets used as this might also trigger people thinking this needs an IDE design review, since we use the same label. Now I'm happy to continue campaigning for the IDE to not use that label and just use a GitHub project for it (since we confusingly have both), but it might be worth making a new "Needs API Review" label for this purpose?

Reply:
Done.